src/asvo/asvo_serde.rs

```diff
@@ -62,8 +62,14 @@ impl DummyJob {
             let mut file_array = vec![];
             for dumb_product in &hm["files"] {
                 file_array.push(AsvoFilesArray {
-                    file_name: dumb_product.file_name.clone(),
-                    file_size: dumb_product.file_size,
+                    r#type: match dumb_product.r#type.as_str() {
+                        "acacia" => Delivery::Acacia,
+                        "astro" => Delivery::Astro,
+                        _ => panic!("Unsupported file type found.")
```

Comment:
Minor point; does this panic emit the delivery type if it's reached? i.e. If "asdf" is received rather than "acacia" or "astro", is "asdf" in the output panic message?

Reply:
It doesn't, but probably should! Good suggestion.